src/config/yaml.ts

```diff
@@ -181,6 +181,9 @@ export const getBuildVersions = (): {
   }
 }
 
+export const getBlacklistedASNs = (): string => yamlConfig?.blacklistedASNs
```

Comment:
this should return an `array`, not a `string`?

Reply:
Correct, thank you. Pushed a fix for this to type as `Array<string>`.